tokio-console/src/warnings.rs

```diff
@@ -150,3 +154,56 @@ impl Warn<Task> for LostWaker {
         "This task has lost its waker, and will never be woken again.".into()
     }
 }
+
+/// Warning for if a task has never yielded
+#[derive(Clone, Debug)]
+pub(crate) struct NeverYielded {
+    min_duration: Duration,
+    description: String,
+}
+
+impl NeverYielded {
+    pub(crate) const DEFAULT_DURATION: Duration = Duration::from_millis(10);
```

Comment:
I think that there are probably many users who have tasks which take more than 10 milliseconds to yield the first time. I would suggest that we are conservative here and not trigger this lint until a task has been running for 1 second and not yielded.

This also matches the default for considering a poll time as an outlier.

```suggestion
    pub(crate) const DEFAULT_DURATION: Duration = Duration::from_secs(1);
```

Reply:
eventually we should really figure out a nice way to add configuration values for lints to the config file...